analytics/index.yaml

```diff
@@ -101,47 +102,46 @@ paths:
           example: http://example.com/path/to/previous/page.html
         - name: _cvar
           in: query
-          #TODO: I don't understand this parameter or it's format. HALP!
-          #TODO: Confirm serialization format (it's supposed to be Array JSON, but example suggests Array-like Object JSON).
           #TODO: Confirm description of item format.
-          #TODO: Add limits of item values (e.g. max length).
-          #TODO: Link to a page describing custom variables.
+          #TODO: Link to a definition of scopes.
           description: |
             <a href="https://help.piwik.pro/analytics/custom-variables/">Custom variables</a> list set on the ``visit``
```

Comment:
To make it clear I would remove `list` from this sentence:
`Custom variables set on the visit scope`.